lib/pure/collections/intsets.nim

```diff
@@ -42,13 +52,17 @@ type
     bits: array[0..IntsPerTrunk - 1, BitScalar] # a bit vector
 
   TrunkSeq = seq[PTrunk]
-  IntSet* = object       ## An efficient set of `int` implemented as a sparse bit set.
+
+  ## A must be an ordinal equivalent type: int | enum | distinct of an int
+  OrdSet*[A] = object    ## An efficient set of `int` implemented as a sparse bit set.
     elems: int           # only valid for small numbers
     counter, max: int
```

Comment:
where is `A` used in type fields?

Reply:
Thank you, fixed

Reply:
if an item is fixed (as opposed to pushback), you can click "resolve conversation" ;-)